RELEASE-NOTES.md

```diff
@@ -10,6 +10,12 @@ The Uberenv project release numbers follow [Semantic Versioning](http://semver.o
 ## Unreleased
 
 ### Added
+- Creates 4 more CI checks to cover more uberenv build options:
+   - dev-build full
+   - dev-build stops at initconfig
+   - install full
+   - install stops at initconfig
```

Comment:
```suggestion
```
We should only add things to this list that users care about and may cause them problems or things they need to know. The other addition is very helpful but we don't want to clutter this list.